src/plugins/raven/services/Raven.js

```diff
@@ -201,6 +210,39 @@ export default class Raven {
     });
   }
 
+  /**
+   * @link https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror
+   *
+   */
+  onError(...args) {
+    const state = this.store.getState();
+    const accessToken = getAccessToken(state);
+    const error = args.find((arg) => arg instanceof Error) || args[0];
+    const logData = {
+      app_version: APP_VERSION,
```

Comment:
@cohencodes also include the current location... like `request_uri: window.location`

Reply:
getting a cors error when trying to send this, need to mess around with it more

Reply:
@cohencodes probably need to replace the `://` here too

Reply:
@efjacobson oh yeah i already tried that it still doesn't like it though

Reply:
@cohencodes if we're ignoring localhost later why do we need to do this replacement at all?

Reply:
@gdbrown i was thinking that too, technically we don't need it, I can remove